content/docs/command-reference/exp/run.md

```diff
@@ -1,8 +1,15 @@
 # exp run
 
-Run or resume a [DVC experiment].
+Run or resume a [DVC experiment] based on a [DVC pipeline].
```

Comment:
We can now present `exp run` as specifically in the context of pipelines, I think (that's the main differentiator vs `exp save` right?)

Reply:
That's correct